test/test_response.py

```diff
@@ -165,6 +165,57 @@ def test_chunked_decoding_gzip(self):
         assert r.read() == b''
         assert r.read() == b''
 
+    def test_decode_gzip_multi_member(self):
+        import zlib
```

Comment:
Let's import zlib at the module scope?

Reply:
No problem.  This is imported many times in the module, do you want me to remove all the similar imports elsewhere in this commit?

Reply:
@isaacto, yes please, unless doing so leads to issues - for example, if we're importing within methods to avoid circular dependencies. If that's the case, then the current arrangement is fine.